TAScheduler/migrations/0002_tatosection_instructortosection.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 4.2.7 on 2023-11-18 05:19
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```

Comment:
I asked ChatGPT: "whether each conceptual table (TA, Instructor, Course) needs a section inside the "operations = []" to perform a "migrations.CreatModel()" operation?" and it said "Yes, each conceptual table (Django model) should have a corresponding migrations.CreateModel operation in the operations list within a migration file" ... I also agree with this statement: I am unsure why you have not defined .CreatModels for all of our tables. Models are blueprints for tables, so it seems strange to not have blueprints for all of them (unless. this commit is not showing all your changes)